DESCRIPTION

```diff
@@ -23,7 +23,7 @@ Description: Create interactive web graphics from 'ggplot2' graphs and/or a cust
 URL: https://plotly-r.com, https://github.com/plotly/plotly.R, https://plotly.com/r/
 BugReports: https://github.com/plotly/plotly.R/issues
 Depends:
-    R (>= 3.2.0),
+    R (>= 3.5),
```

Comment:
Is this necessary?

Reply:
The line you're referring to is a build time, not run time dependency, so the version bump requirement isn't necessary

Reply:
Yeah, but it also becomes a *run*time dependency once we ship pkg-internal data in v3 format. To cite the [relevant doc](https://usethis.r-lib.org/reference/use_data.html#arg-version):

> Version 3 became the default from R 3.6.0 and **can only be read by R versions 3.5.0 and higher**.

Reply:
Ahh, right, thanks for pointing that out. Could we just keep the current behavior? It doesn't seem worth it to me to break compatibility for relatively small R objects

Reply:
Keeping the current behaviour is fine with me. But OTOH, R 3.5.0 was [released more than 6 years ago](https://stat.ethz.ch/pipermail/r-announce/2018/000628.html), so I don't think we ought to care about supporting more ancient versions.

I'll have a look at your other comments tomorrow. I'll revert to v2 serialization format then if you still think it's worth it :)

Reply:
I've reverted to the v2 serialization format, see https://github.com/plotly/plotly.R/pull/2336/commits/93ef777a672852a2aa48c9f7ed5c4c2f995713b5.